lib/src/Uncertainty/Algorithm/Analytical/Analytical.cxx

```diff
@@ -131,6 +130,19 @@ void Analytical::run()
   Point value(standardEvent.getImplementation()->getFunction().operator()(origin));
 
   result_.setIsStandardPointOriginInFailureSpace(event_.getOperator().compare(value[0], event_.getThreshold()));
+  
+  /* check if result is valid */
+ const Point physicalSpaceDesignPoint(event_.getImplementation()->getAntecedent().getDistribution().getInverseIsoProbabilisticTransformation().operator()(standardSpaceDesignPoint));
+  
+  const Point valuePhysicalSpaceDesignPoint(event_.getImplementation()->getFunction().operator()(physicalSpaceDesignPoint));
```

Comment:
this should be equal to nearestPointAlgorithm_.getResult().getOptimalValue(), thus avoiding an extra evaluation